taca/utils/bioinfo_tab.py

```diff
@@ -143,7 +158,7 @@ def get_ss_projects(run_dir):
                 FCID_samplesheet_origin = os.path.join(run_dir,'SampleSheet.csv')
                 if not os.path.exists(FCID_samplesheet_origin):
                     logger.warn("Cannot locate the samplesheet for run {}".format(run_dir))
-                    return ['UNKNOWN']
+                    return []
```

Comment:
@sylvinite, you see what I mean by forcing UNKNOWN ? :P

On Tue, Dec 1, 2015 at 11:48 AM, Francesco <notifications@github.com> wrote:

> In taca/utils/bioinfo_tab.py
> <https://github.com/SciLifeLab/TACA/pull/149#discussion_r46264011>:
>
> > @@ -143,7 +158,7 @@ def get_ss_projects(run_dir):
> >                  FCID_samplesheet_origin = os.path.join(run_dir,'SampleSheet.csv')
> >                  if not os.path.exists(FCID_samplesheet_origin):
> >                      logger.warn("Cannot locate the samplesheet for run {}".format(run_dir))
> > -                    return ['UNKNOWN']
> > +                    return []
>
> I have a rule that redirects seqmaster mails to a folder and marks them as
> read....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/TACA/pull/149/files#r46264011>.
>